scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -358,7 +360,7 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
     {% else %}
       No pinned variants available for this case
     {% endif %}
-    {% if variants.suspects_detailed and duplicated_variants|length == variants.suspects_detailed.suspects_detailed|length %}
+    {% if pinned_not_causative_variants|length == 0 %}
```

Comment:
Yep, more direct and actually correct. 😄

Reply:
Ah, hm, well, actually checking for any pinned would be pretty good as well, or you would end up with No pinned vars - All pinned vars are causative..